doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md

```diff
@@ -5,4 +5,366 @@ permalink: /:path/
 sort_idx: 20
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+## Introduction
+
+This page will guide you through Kaa C SDK installation and compilation process for the ESP8266 platform.
+All steps described here were tested on *Linux Ubuntu 14.04 x86_64*.
+
+## Connecting ESP8266
+
+To flash ESP8266 chip you need to connect it to your PC. That may differ depending on the module you use.
+This guide covers instructions for ESP8266-01 and NodeMCU modules.
+
+### ESP8266-01
+
+To connect the ESP8266-01 to PC, a 3.3V USB-to-TTL connector is required.
+A 3.3V USB-to-TTL connector is required to connect the ESP8266-01 to PC.
+There are two boot modes for ESP8266: flash mode and run mode.
+The table below summarizes wiring scheme for both boot modes.
+
+<ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#Flash">Fash Mode</a></li>
+    <li><a data-toggle="tab" href="#Run">Run mode</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="Flash" class="tab-pane fade in active" markdown="1" >
+
+#### Flash mode
```

Comment:
Is it necessary to add headers for "Flash mode" and  "Run mode"? "Run mode" is not clickable in Toc, if "Flash mode" tab is active.